cfpb/cfgov-refresh
# Mega menu mobile: Consolidate scroll to top code

We were scrolling to the top more than once in some cases (thanks @jimmynotjim for the find). This consolidates those. Not sure if `window.scrollTo( 0, 0 )` or `document.body.scrollTop = document.documentElement.scrollTop = 0` is better, but went with the latter.

## Changes

- Consolidates scroll to top behavior on mobile mega menu to top of on click handler.

## Testing

1. Visit consumerfinance.gov on iOS 11
2. Navigate to About Us
3. Scroll down so the top of the menu is off screen.
4. Click Careers.
5. Scroll should return to the top of the screen.
